docs/app/Examples/elements/Input/Variations/InputRightLeftLabeled.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import { Input, Label } from 'stardust'
+
+const InputRightLeftLabeled = () => (
+  <Input labelPosition='right' placeholder='mysite.com'>
```

Comment:
I wish :(  The input has some very odd markup.  I think it needs some core SUI work to make more sense.  The `labeled` class alone formats the left label.  While adding `right labeled` also formats the right label, but the `labeled` portion of that class still formats the left label also.  Very confusing and unfortunate. 

http://semantic-ui.com/elements/input.html#labeled

![image](https://cloud.githubusercontent.com/assets/5067638/18924302/86d2d3c4-8564-11e6-89be-72debd00ebd1.png)